File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */

var findNumbers = function (nums) {
    let counter = 0;
    for (let i = 0; i <= nums.length-1; i++) {
        (nums[i].toString().length % 2 === 0) ? counter+=1 : null;

    }
    return counter;
};

File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        //initialize a counter
        int counter=0;
        
        //loop over the whole array elements
        for(int i=0;i<nums.size();i++){
            //convert the integer to string first, then check its length is even or not through the modulus division
            if((to_string(nums[i]).size()%2)==0)
                counter++;
        }
        
        //return the counter value
        return counter;
    }
};
